### ISSUE TITLE
package or namespace load failed for 'taxize'

### ISSUE THREAD

**Issue by cywhale** [jeroenooms/opencpu#211]
The package 'taxize' cannot be loaded in my self-written packages (both Depends or Imports cannot) through Opencpu API. But It work well when load the library under Rstudio.
> find.package("taxize")
[1] "/usr/local/lib/R/site-library/taxize"

When I use curl to call my functions, it just showed the following message: "package or namespace load failed for 'taxize' ". I googled some similar problems happened just like "package can loaded in Rstudio, but cannot loaded through opencpu", but still cannot solve my problem. 

I cannot figure out what's wrong in this issue. Could you help, or any comments? 

$ curl -L http://localhost/ocpu/library/taxize/R
package or namespace load failed for 'taxize'

$ curl -L http://localhost/ocpu/library/taxize
		Information on package 'taxize'
Description:
Package:            taxize
Title:              Taxonomic Information from Around the Web
Description:        Interacts with a suite of web 'APIs' for taxonomic
                    tasks, such as verifying species names, and getting
                    taxonomic 'hierarchies'.
Version:            0.7.9
....

**Comment by jeroenooms** [comment]
What OS are you on?

**Comment by cywhale** [comment]
Ubuntu 14.04
######## here is my environment in Rstudio
> sessionInfo()
R version 3.3.2 (2016-10-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

locale:
 [1] LC_CTYPE=en_PH.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_PH.UTF-8        LC_COLLATE=en_PH.UTF-8    
 [5] LC_MONETARY=en_PH.UTF-8    LC_MESSAGES=en_PH.UTF-8   
 [7] LC_PAPER=en_PH.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_PH.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets 
[6] methods   base     

other attached packages:
[1] odbapi_0.42       taxize_0.7.9      data.table_1.9.6 
[4] RPostgreSQL_0.4-1 DBI_0.5-1        

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.8      xml2_1.0.0       magrittr_1.5    
 [4] rotl_3.0.1       ape_3.5          lattice_0.20-34 
 [7] R6_2.2.0         foreach_1.4.3    stringr_1.1.0   
[10] rentrez_1.0.4    httr_1.2.1       plyr_1.8.4      
[13] tools_3.3.2      rredlist_0.2.0   bold_0.3.5      
[16] grid_3.3.2       nlme_3.1-128     iterators_1.0.8 
[19] assertthat_0.1   reshape2_1.4.2   codetools_0.2-15
[22] rsconnect_0.6    stringi_1.1.1    rncl_0.6.0      
[25] XML_3.98-1.1     jsonlite_1.1     reshape_0.8.6   
[28] chron_2.3-47

**Comment by jeroenooms** [comment]
Can you check the log files `/var/log/apache/error.log` and `/var/log/opencpu/error.log` and `/var/log/kern.log` if they contain any messages that suggest while the load has failed?

**Comment by cywhale** [comment]
in kern.log, repeated messages like:
Nov 22 20:53:16 odbadmin-virtual-machine kernel: [7190533.376050] audit: type=1400 audit(1479819196.598:386): apparmor="STATUS" operation="profile_replace" profile="unconfined" name="r-base" pid=18782 comm="apparmor_parser"

No messages in the other two error.log
Thanks

**Comment by oliverbock** [comment]
If you are running from source, consider commenting out the two lines either side of

 response <- serve(REQDATA);

in rapachehandler.R, because these can prevent errors from appearing in Apache's error.log.  (I had a problem with a dying apparmor subprocess that was easy to solve once I did this - related to some bad code in a package, which I could work around.)

**Comment by cywhale** [comment]
I just use opencpu or rapache from installation, not from source. I can try it later. Indeed, my older versions work well with taxize or other packages I imported through Opencpu API. I thought it's a version issue, but when I installed my old versions, including original taxize versions, it still failed. So maybe there're some changes in running environments I don't know (through apt-get update or some others..) make this problem.
